### DISCUSSION TITLE
Replacing submitted model

### DISCUSSION THREAD

**Discussion by YutackPark** [janosh/matbench-discovery#168] [Q&A] [answered]
Dear devs

We're preparing new SevenNet modal submission that is Matbench compliant (trained on MPtrj + open code + open data). Currently, the model column of Matbench leaderboard is a little bit confusing to me. Some of them are general model architecture or package name (MACE instead of MACE-MP-0) and others have suffixes about pre-trained model (eqV2 S DeNS instead of eqV2).For SevenNet, more precise name is SevenNet-0.

Could you guide me how should name if we submit a new SevenNet modal? 

Plus, should we test kSRME for the submission? (I already have prepared relaxed structures!)

**Comment by janosh** [comment, marked as answer]
Hi @YutackPark! good point, we should be more specific and rename MACE to MACE-MP-0 and SevenNet to SevenNet-0.

> Could you guide me how should name if we submit a new SevenNet modal?

you get to pick the name. just needs to be unique. so SevenNet-1 might make sense.

> Plus, should we test kSRME for the submission?

yes please! have a look at [`models/SevenNet/1_test_srme.py`](https://github.com/MPA2suite/k_SRME/blob/main/models/SevenNet/1_test_srme.py). also please let us know if you notice anything that could be improved about how SevenNet-0 was tested. pinging @bpota and @MSimoncelli for context
